packages/devtools-local-toolbox/webpack.config.devtools.js

```diff
@@ -21,15 +21,22 @@ const nativeMapping = {
 };
 
 let packagesPath = path.join(__dirname, "../");
+let projectPath = path.join(__dirname, "../../");
 
 module.exports = webpackConfig => {
   webpackConfig.output.library = "Debugger";
 
   if (process.env.MOCHITESTS) {
-    webpackConfig.output.path = path.join(__dirname, "firefox/devtools/client/debugger/new");
+    webpackConfig.output.path = path.join(
+      projectPath,
+      "firefox/devtools/client/debugger/new"
+    );
```

Comment:
fixes an issue where `npm run mochitest-watch" was placing the build in *devtools-local-toolbox/firefox*